src/NServiceBus.AcceptanceTests/ApiExtension/When_extending_sendoptions.cs

```diff
@@ -12,11 +13,9 @@
     public class When_extending_sendoptions : NServiceBusAcceptanceTest
     {
         [Test]
-        public void Should_be_able_to_set_context_items_and_retrieve_it_via_a_behavior()
+        public async Task Should_be_able_to_set_context_items_and_retrieve_it_via_a_behavior()
         {
-            var context = new Context();
-
-            Scenario.Define(context)
+            var context = await Scenario.Define<Context>()
```

Comment:
It is less important. I think it is ok as long we do it internally in the framework and the ATT test library